frontend/src/components/account-settings.ts

```diff
@@ -225,15 +224,15 @@ export class AccountSettings extends LiteElement {
                     label=${msg("Enter your current password")}
                     type="password"
                     autocomplete="current-password"
-                    password-toggle
+                    passwordToggle
```

Comment:
From https://shoelace.style/components/input, it seems like we want the attribute and not the property here, so should be `password-toggle` I think?